src/Raven.Studio/typescript/common/shell/accessManager.ts

```diff
@@ -110,51 +108,69 @@ class accessManager {
-                case "ClusterAdmin":
-                case "ClusterNode":
-                    return effectiveDatabaseAccess === "DatabaseAdmin";
-            }
-        } else {
-            switch (requiredAccess) {
-                case "Operator":
-                    return this.isOperatorOrAbove();
-                case "ClusterAdmin":
-                case "ClusterNode":
-                    return this.isClusterAdminOrClusterNode();
-                default:
-                    throw new Error("RequiredAccess for a 'Server View' must be either: ClusterAdmin, ClusterNode, Operator");
-            }
+        const clusterAdminOrNode = actualAccessLevel === "ClusterAdmin" || actualAccessLevel === "ClusterNode";
+        const operator = actualAccessLevel === "Operator";
+        const dbAdmin = actualAccessLevel === "DatabaseAdmin";
+        const dbReadWrite = actualAccessLevel === "DatabaseReadWrite";
+        const dbRead = actualAccessLevel === "DatabaseRead";
+        
+        switch (requiredAccess) {
+            case "ClusterAdmin":
+            case "ClusterNode":
+                return clusterAdminOrNode;
+            case "Operator":
+                return clusterAdminOrNode || operator;
+            case "DatabaseAdmin":
+                return clusterAdminOrNode || operator || dbAdmin;
+            case "DatabaseReadWrite":
+                return clusterAdminOrNode || operator || dbAdmin || dbReadWrite;
+            case "DatabaseRead":
+                return clusterAdminOrNode || operator || dbAdmin || dbReadWrite || dbRead;
+            default: 
+                return false;
         }
     }
     
-    getDisableReasonHtml(dbName: string, requiredAccess: accessLevel, effectiveDbAccess: databaseAccessLevel) {
-        const title = dbName ? "Insufficient database access" :
-                               "Insufficient security clearance";
-        
+    getDisableReasonHtml(requiredAccess: accessLevel, actualAccessLevel: accessLevel) {
         const requiredText = this.getAccessLevelText(requiredAccess);
         
-        const actualText = dbName ? this.getAccessLevelText(effectiveDbAccess) :
-                                    this.getAccessLevelText(this.securityClearance());
-        
-        return `<div class="text-left">
-                    <h4>${title}</h4>
+        if (actualAccessLevel) {
+            const actualText = this.getAccessLevelText(actualAccessLevel);
+
+            return `<div class="text-left">
+                    <h4>Insufficient access</h4>
                     <ul>
                         <li>Required: <strong>${requiredText}</strong></li>
                         <li>Actual: <strong>${actualText}</strong></li>
                     </ul>
                 </div>`;
+        } else {
+            return `<div class="text-left">
+                    <h4>Insufficient access</h4>
+                    <ul>
+                        <li>Required: <strong>${requiredText}</strong></li>
```

Comment:
IMHO - I prefer the previous tooltips version
This is Not informative enough

Reply:
done

Reply:
ok